src/content/tutorials/en/setting-up-your-environment.mdx

```diff
@@ -4,7 +4,7 @@ description: A quick tutorial for setting up the p5.js Web Editor and VS Code to
 category: introduction
 categoryIndex: 0
 featuredImage: ../images/featured/SetupA.jpg
-featuredImageAlt: An interactive sketch in the p5.js Web Editor draws circles on the canvas as the mouse pointer moves.
+featuredImageAlt: An interactive sketch that draws circles as the mouse pointer moves.
```

Comment:
Seems like "An interactive sketch in the p5.js Editor draws circles on the canvas as the mouse pointer moves." would also be under 100 characters, and preferred, as it keeps more of the information